src/services/Firebase.jsx

```diff
@@ -25,20 +25,50 @@ import {
 import { v4 as uuid4 } from "uuid";
 import { compareSync, hash, hashSync } from "bcryptjs";
 
+/**
+ * Firebase configuration object.
+ * @type {Object}
+ */
 const firebaseConfig = {
-  apiKey: "AIzaSyCeCiujs2TwwPw6GHBhsmA5NATqtzk_wqo",
-  authDomain: "sustainify-4ee8b.firebaseapp.com",
-  projectId: "sustainify-4ee8b",
-  storageBucket: "sustainify-4ee8b.appspot.com",
-  messagingSenderId: "1084970092267",
-  appId: "1:1084970092267:web:09da9473889c4c2dcaa0ae",
+  apiKey: "YOUR_API_KEY",
+  authDomain: "YOUR_AUTH_DOMAIN",
+  projectId: "YOUR_PROJECT_ID",
+  storageBucket: "YOUR_STORAGE_BUCKET",
+  messagingSenderId: "YOUR_MESSAGING_SENDER_ID",
+  appId: "YOUR_APP_ID",
```

Comment:
This github repo is needed for deploying the app, we can't just put these dummy strings here, leave the API key as it is, once the project is completed and working, I will put these in an .env file